db/migrate/20200802222331_create_payments.rb

```diff
@@ -0,0 +1,18 @@
+class CreatePayments < ActiveRecord::Migration[6.0]
+  def change
+    create_table :payments, id: :uuid do |t|
+      t.date :paid_on, null: false
+      t.date :care_started_on, null: false
+      t.date :care_finished_on, null: false
+      t.monetize :amount, null: false, default: 0
+      t.string :slug, null: false
+      t.monetize :discrepancy, default: 0
```

Comment:
Can we remove this default so we can retain a null state if the discrepancy hasn't been set at all?  Also I wonder if setting this default is what's marking `discrepancy_cents` and `discrepancy_currency` as `not null` in the schema. 🤔 If you remove this default, can you do `rails annotate_models` and see if these are updated?